src/Specs/model.json

```diff
@@ -53,23 +64,34 @@
             "type": "object",
             "description": "Model field definitions",
             "additionalProperties": {
-                "type": "string",
-                "description": "Field data type",
-                "enum": [
-                    "ulid",
-                    "id",
-                    "uuid",
-                    "string",
-                    "integer",
-                    "boolean",
-                    "date",
-                    "datetime",
-                    "decimal",
-                    "text",
-                    "json",
-                    "jsonb",
-                    "object"
-                ]
+                "type": "object",
+                "properties": {
+                    "type": {
+                        "type": "string",
+                        "description": "Field data type",
+                        "enum": [
+                            "uuid",
+                            "string",
+                            "timestamp",
```

Comment:
@rajeshn95 What's diff between timestamp and datetime? Do we need timestamp?

Reply:
@rjvim we will need only timestamp, this I have to clean up.